gammapy/cube/tests/test_fit.py

```diff
@@ -0,0 +1,90 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import pytest
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord, Angle
+from astropy.tests.helper import assert_quantity_allclose
```

Comment:
Change `assert_quantity_allclose` to the version in `gammapy.utils.testing` (a change I made a day ago everywhere else).